doc/CHANGES-staging/app_queue.txt

```diff
@@ -0,0 +1,9 @@
+Subject: app_queue
```

Comment:
There are no longer CHANGES files, this should be a UserNote.

Reply:
Oops.  This was forward-ported from a previous attempt at this patch which got stuck on gerrit.